docs/advanced-features/using-mdx.md

```diff
@@ -191,7 +191,7 @@ import Image from 'next/image'
 import { Heading, InlineCode, Pre, Table, Text } from 'my-components'
 
 const ResponsiveImage = (props) => (
-  <Image alt={props.alt} layout="responsive" {...props} />
+  <Image alt={props.alt} fill {...props} />
```

Comment:
```suggestion
  <Image alt={props.alt} sizes="100vw" style={ width: '100%', height: 'auto' }  {...props} />
```

Reply:
Ah yes, my bad! Indeed this would be the correct up to date approach 😅 Thanks